dashboard/app/controllers/learning_goal_evaluations_controller.rb

```diff
@@ -0,0 +1,38 @@
+class LearningGoalEvaluationsController < ApplicationController
+  authorize_resource
+
+  def create
+    learning_goal_evaluation = LearningGoalEvaluation.new(learning_goal_evaluation_params)
+    learning_goal_evaluation.teacher_id = current_user.id
+
+    learning_goal_evaluation.save!
+
+    render json: learning_goal_evaluation
+  end
+
+  def update
+    params.require(:id)
+    learning_goal_evaluation = LearningGoalEvaluation.find_by_id(params[:id])
+
+    if learning_goal_evaluation.teacher_id == current_user.id
+      learning_goal_evaluation.update(learning_goal_evaluation_params)
+      render json: learning_goal_evaluation
+    else
+      render status: :not_found, json: learning_goal_evaluation
```

Comment:
Please don't say you can't find the record and then return the record at the same time. :-)

```suggestion
      head :not_found
```

Reply:
Haha good catch. Thanks!